src/Components/Share/Share.tsx

```diff
@@ -49,6 +54,14 @@ const Share = forwardRef(function Share() {
     }, 5000);
   };
 
+  const onEmailClick = () => {
+    window.location.href = email;
```

Comment:
Would love feedback from @sydney-devine as well, but I believe this one should have the same expected behavior as a mailto: protocol

Reply:
@CalebPena or @Mdemenko1 is there a video of what the behavior is right now? 

From memory, one was supposed to link to a like feedback survey type deal, and that should def open in a new tab. Sorry, do we also have the finalized mockup that we were going off of linked around here somewhere? Trying to refresh my memory on this.

Reply:
Ack sorry. I scrolled up and saw the video. 

The report a bug right now is opening in a new tab. So that seems like the right expected behavior. @CalebPena are you seeing something else?

Contact us I think should do like a mailto: function